experiments/modern/src/maki-interpreter/virtualMachine.js

```diff
@@ -134,16 +144,34 @@ async function interpret(start, program, { logger = null }) {
         stack.push(obj[methodName](...methodArgs));
         break;
       }
+      // callGlobal
+      case 25: {
+        // This is where the version checked wa5 scripts start with this offset
+        if (command.arguments[0].offset === 4294967296) {
```

Comment:
Oh. There's a bug in my code that's causing this number. The decompiler reads a _signed_ 32 bit int when parsing this offset, but I'm reading it as _unsigned_. If I recall correctly that results in this value being 0. We can merge this as is and I'll clean it up when I add the ability to get signed ints from the parser.

Reply:
haha, now that you mention it I remember you mentioning that in discord. It probably just runs the version check, although we might wanna skip those anyway. Easy enough to test again when that changes, thanks